syntax = "proto3";

package registration;

service Registration {
    rpc registerUser (RegisterRequest) returns (RegisterResponse);
}

message RegisterRequest {
    string username = 1;
    string email = 2;
    string password = 3;
}

message RegisterResponse {
    bool success = 1;
}